serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -13,14 +13,11 @@ After events have been sent to a channel, these events can be sent to multiple K
 The default configuration for channel instances is defined in the `default-ch-webhook` ConfigMap. However, developers can still create their own channels directly by instantiating a supported channel object.
 
```

Comment:
Any prerequisites for creating channels? Other components we need first, e.g. broker?